eos/b-decays/lifetime.cc

```diff
@@ -272,6 +273,190 @@ namespace eos
+                },
+                {
+                    -60.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    -816.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    -60.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -144.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    768.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z))
+                },
+                {
+                    -6.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -60.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -3.0 / 2.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    0,
+                    -60.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                },
+                {
+                    -36.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -144.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    0,
+                    -36.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    576.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                },
+                {
+                    336.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    768.0 * z * (-1.0 - 9.0 * std::pow(z, 2) + 9.0 * std::pow(z, 4) + std::pow(z, 6) - 12.0 * (std::pow(z, 2) + std::pow(z, 4)) * log(z)),
+                    -60.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    576.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z)),
+                    -12480.0 * (-1.0 + 8.0 * std::pow(z, 2) - 8.0 * std::pow(z, 6) + std::pow(z, 8) + 24.0 * std::pow(z, 4) * log(z))
+                }
+            }};
+
+            const std::array<std::complex<double>, 5u> C_sing
```

Comment:
I would order this as ``C_sing`` before ``G``. A few brief comment on what the various quantities are would be appreciated.

Reply:
Also, can you elaborate why you append the suffix ``_sing``?